backend/src/test/java/ai/verta/modeldb/ProjectTest.java

```diff
@@ -834,7 +834,7 @@ public void ffff_addProjectTagNegativeTest() {
     } catch (StatusRuntimeException ex) {
       Status status = Status.fromThrowable(ex);
       LOGGER.warn("Error Code : " + status.getCode() + " Description : " + status.getDescription());
-      assertEquals(Status.INVALID_ARGUMENT.getCode(), status.getCode());
+      assertEquals(Status.PERMISSION_DENIED.getCode(), status.getCode());
```

Comment:
With new changes, we are checking permission check first so from there we are throwing PERMISSION_DENIED.